contrib/pcsc-ccid/template.py

```diff
@@ -0,0 +1,20 @@
+pkgname = "pcsc-ccid"
+pkgver = "1.5.5"
+pkgrel = 0
+build_style = "gnu_configure"
+configure_args = ["--enable-twinserial"]
+configure_gen = []
```

Comment:
i assume it fails to reconf it?

Reply:
```
=> ccid-1.5.5-r0: running do_configure...
libtoolize: putting auxiliary files in AC_CONFIG_AUX_DIR, '.'.
libtoolize: copying file './ltmain.sh'
libtoolize: putting macros in AC_CONFIG_MACRO_DIRS, 'm4'.
libtoolize: copying file 'm4/libtool.m4'
libtoolize: copying file 'm4/ltoptions.m4'
libtoolize: copying file 'm4/ltsugar.m4'
libtoolize: copying file 'm4/ltversion.m4'
libtoolize: copying file 'm4/lt~obsolete.m4'
configure.ac:36: error: possibly undefined macro: AC_MSG_ERROR
      If this token and others are legitimate, please use m4_pattern_allow.
      See the Autoconf documentation.
configure.ac:136: error: possibly undefined macro: AC_DEFINE
autoreconf: error: /usr/bin/autoconf failed with exit status: 1
=> A failure has occurred!
```

Reply:
yea amazing, thanks for checking

Reply:
Is there a problem with having `configure_gen = []` in there?

Reply:
(I don't know what it does, I've cargo-culted it from somewhere but don't remember where)

Reply:
no (unless it's broken without regen, which happens with some old ones, due to old autoconf generating conftests that new clang fails on unconditionally), i was just curious (sometimes people copypaste it from another template but it reconfs fine)

Reply:
> due to old autoconf   

(any version before 2.72, technically, though most distros backported patches)

Reply:
ack, okay. Anything else left to do here then?

Reply:
for this, all good :)